velox/exec/tests/TableWriteTest.cpp

```diff
@@ -758,7 +758,7 @@ class TableWriteTest : public HiveConnectorTestBase {
 
   // Verifies the bucketed file data by checking if the bucket id of each read
   // row is the same as the one encoded in the corresponding bucketed file name.
-  void verifyBucketedFileData(const std::filesystem::path& filePath) {
+  void verifyBucketedSortedFileData(const std::filesystem::path& filePath) {
```

Comment:
Let's just call it verifyBucketedFileData as sorting is optional property to verify internally.